openghg/store/_footprints.py

```diff
@@ -350,39 +349,13 @@ def read_file(
             return {}
 
         # Do some housekeeping on the inputs
-        if species == "co2":
-            # Expect co2 data to have high time resolution
-            if not time_resolved:
-                logger.info("Updating time_resolved to True for co2 data")
-                time_resolved = True
+        time_resolved = check_species_time_resolved(species, time_resolved)
+        short_lifetime = check_species_lifetime(species, short_lifetime)
 
-            if sort:
-                logger.info(
-                    "Sorting high time resolution data is very memory intensive, we recommend not sorting."
-                )
-
-        if short_lifetime:
-            if species == "inert":
-                raise ValueError(
-                    "When indicating footprint is for short lived species, 'species' input must be included"
-                )
-        else:
-            if species == "inert":
-                lifetime = None
-            else:
-                lifetime = species_lifetime(species)
-                if lifetime is not None:
-                    # TODO: May want to add a check on length of lifetime here
-                    logger.info("Updating short_lifetime to True since species has an associated lifetime")
-                    short_lifetime = True
-
-        chunks = self.check_chunks(
-            filepaths=filepath,
-            chunks=chunks,
-            high_spatial_resolution=high_spatial_resolution,
-            time_resolved=time_resolved,
-            short_lifetime=short_lifetime,
-        )
+        if time_resolved and sort:
```

Comment:
As far as I understand it, `open_mfdataset` will sort automatically. So if multiple files are passed, they will be sorted by `open_mfdataset` and if one file is passed, there is no need to sort it. So the `sort` option and this warning should probably be removed.

Reply:
This all links to the fact that `sort` is used within `Datasource.add_timed_data` function 

```
225        if sort and drop_duplicates:
226            data = data.drop_duplicates(time_coord, keep="first").sortby(time_coord)
227        elif sort:
228            data = data.sortby(time_coord)
```

so the presence of `sort` does mean an action is being performed. That action may not be necessary so potentially could be removed but I wouldn't want to do that in this PR. In which case I do think the warning is telling us something at the moment.

Reply:
I opened an issue for this: https://github.com/openghg/openghg/issues/1081. In this case, we know that the data either consists of one dataset or was opened by `open_mfdataset`, so we could pass `sort=False` to assign data... anyway we don't have to do this now.